source/Projectile.cpp

```diff
@@ -61,8 +61,8 @@ Projectile::Projectile(const Ship &parent, Point position, Angle angle, const We
 
 
 
-Projectile::Projectile(const Projectile &parent, const Weapon *weapon)
-	: Body(weapon->WeaponSprite(), parent.position + parent.velocity, parent.velocity, parent.angle),
+Projectile::Projectile(const Projectile &parent, const Point &offset, const Angle &angle, const Weapon *weapon)
+	: Body(weapon->WeaponSprite(), parent.position + parent.velocity + offset, parent.velocity, parent.angle + angle),
```

Comment:
The new offset is used as an absolute offset, without considering the orientation of the projectile. Is that intended?
If not, it could be addressed here or at the sites where this constructor gets called.

Reply:
No, that was not intended; the orientation of the projectile should be considered. I'll fix that.